### ISSUE TITLE
Kolbot issue with home+ space

### ISSUE THREAD

**Issue by dverial** [kolton/d2bot-with-kolbot#986]
Here it is, I'm trying to congig my char for mf finds, but the home button+ space is not working, the buttons are working fine but it seems they do not work when game is up and running! Help please I need to configurate that char 
Thanks!

**Comment by DarkHorseDre** [comment]
what happens when the bot creates a game and toon arrives in town?

what happens when you then press home once?

**Comment by dverial** [comment]
I'm in town, everything seems fine, but when I press absolutly nothing happens. it just makes a space in the commande section at the top. Really cant figure out whats wrong

**Comment by DarkHorseDre** [comment]
ok, not sure what you mean.
please post a screenshot showing the screen when game starts and the screen after pressing home

**Comment by dverial** [comment]
yes I'm on a laptop and I had issues with the keyboard in the past. I can try to give u a screen shot. But like I said, nothing happens, the window tab opens up but in game nothing

**Comment by dverial** [comment]
solved it, its an issue with Lenovo legion y520. Thx anyway :)

**Comment by DarkHorseDre** [comment]
lol yup :)

fyi - see that > on the last line? its a prompt.
press ALT+HOME and it will disappear (else everything you type will be on that command line.  "reload" will force bot to reload scripts without exiting game. handy if you've changed the config/pickit on the fly).

then home and space will work as expected/you can type as expected in-game

**Comment by Mysticos05** [comment]
I'm still having this same issue... ALT+HOME didn't work to clear the prompt. And HOME+SPACE doesn't work. I'm using a Microsoft Surface Laptop

**Comment by DarkHorseDre** [comment]
I've never used the surface - try the onscreen keyboard - does alt+home work?

home is probably the issue with a laptop keyb. do you need to press a function key to get home to work?
